src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,39 +1,83 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
+
+//components
+
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo;
+  setSelectedTodo: () => void;
```

Comment:
The `setSelectedTodo` prop should be a function that accepts a `Todo` object or `null`, not a function that returns void. Update the type to `(todo: Todo | null) => void`.